src/components/Message.jsx

```diff
@@ -0,0 +1,20 @@
+import React, {PureComponent} from "react"
+
+class Message extends PureComponent {
+
+    render() {
+        return (
+            <>
+                <p style={style}> {this.props.message} </p>
+            </>
+        )
+    }
+}
+
+const style = {
```

Comment:
@CharlotteAins @gvozdev1986 
question for you: why does it work? :) https://github.com/gvozdev1986/ReactJS-Global-Mentoring-Program-2020Q3/pull/2/files#diff-dd7c0af8479e4f34d3023befc96348eeR8

Reply:
As I understand it, we can use <React.Fragment> to avoid unnecessary div and <React.Fragment> can create a list of child elements, without creating unnecessary nodes in the DOM.